filclient.go

```diff
@@ -1322,7 +1318,7 @@ func (fc *FilClient) CheckChainDeal(ctx context.Context, dealid abi.DealID) (boo
 func (fc *FilClient) CheckOngoingTransfer(ctx context.Context, miner address.Address, st *ChannelState) (outerr error) {
 	defer func() {
 		// TODO: this is only here because for some reason restarting a data transfer can just panic
-		// https://github.com/filecoin-project/go-data-transfer/issues/150
+		// https://github.com/filecoin-project/go-data-transfer/v2/issues/150
```

Comment:
```suggestion
		// https://github.com/filecoin-project/go-data-transfer/issues/150
```

Reply:
the https url is unchanged. yay go modules being weird.